client/blocks/plan-thank-you-card/style.scss

```diff
@@ -0,0 +1,226 @@
+.plan-thank-you-card {
+	position: relative;
+	width: 100%;
+	max-width: 500px;
+	background-color: darken( $alert-green, 10 );
+	border-radius: 8px 8px 6px 6px;
+	overflow: hidden;
+	margin: 56px auto 24px;
+	color: $white;
+	font-family: $sans;
+	text-align: center;
+
+	@media ( max-width: 500px ) {
+		margin-top: 0;
+	}
+}
+
+.plan-thank-you-card__header {
+	background-color: $alert-green;
+	padding: 75px 0 30px;
+	position: relative;
+	z-index: 1;
```

Comment:
Again, this needs to be added to the z-index map. I would suggest avoiding any z-index rules if possible. We should be able to order most things using the [natural stacking order](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Positioning/Understanding_z_index/Stacking_without_z-index).


https://github.com/Automattic/wp-calypso/blob/master/docs/coding-guidelines/css.md#z-index